docs/about/index.rst

```diff
@@ -1,32 +1,198 @@
 .. _about:
 
-About
-=====
+About Fairlearn
+===============
+
+Fairlearn is a project of the
+`Fairlearn Organization <https://github.com/fairlearn/governance/blob/main/ORG-GOVERNANCE.md>`_
+and follows the
+`Fairlearn Organization's Project Governance <https://github.com/fairlearn/governance/blob/main/PROJECT-GOVERNANCE.md>`_.
 
 .. _mission:
 
 Mission
 -------
 
+Fairlearn is an open-source, community-driven project to help data scientists
+improve fairness of AI systems.
+
+The project aspires to include:
+
+- A Python library for fairness assessment and improvement (fairness metrics, 
+  mitigation algorithms, plotting, etc.)
+- Educational resources covering societal and technical processes for 
```

Comment:
@hildeweerts I see you added this part. 😄 Are there really "societal [...] processes"? I'm not sure that would make sense to me as a reader.

Reply:
I copied that from the homepage of the website but wasn't too sure about it either. Do you have a suggestion? I was thinking something like business/development processes but didn't feel like that captured the message either...

Reply:
Maybe organizational processes? @romanlutz wdyt?